server/e2e_test_command.go

```diff
@@ -0,0 +1,253 @@
+package server
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/mattermost/mattermost-mattermod/model"
+	"github.com/mattermost/mattermost-server/v5/mlog"
+)
+
+const (
+	e2eTestMsgCommenterPermission = "Looks like you don't have permissions to trigger this command.\n Only available for org members"
+	e2eTestMsgCIFailing           = "Command e2e-test requires PR checks to pass."
+	e2eTestMsgUnknownPRState      = "Failed to check if PR checks passed. I will not try to trigger e2e testing. Please retry in a bit."
+	e2eTestMsgPRInfo              = "Failed to get the PR information required to trigger testing. Please retry in a bit."
+	e2eTestMsgUnknownTriggerRepo  = "Looks like triggering e2e testing pipelines for this repo is not set up. \n Please contact your administrator of choice."
```

Comment:
```suggestion
	e2eTestMsgUnknownTriggerRepo  = "The ability to triggering E2E testing pipelines for this repository isn't set up. \n Please contact your system admin."
```

Reply:
Should it then be 
```suggestion
	e2eTestMsgUnknownTriggerRepo  = "The ability to trigger E2E testing pipelines for this repository isn't set up. \n Please contact a maintainer."
```

Reply:
Thanks for catching my typo! I agree with your updates.